tests/Sample.elm

```diff
@@ -0,0 +1,11 @@
+module Sample exposing (..)
```

Comment:
@zacck Looks good. Please run [elm-format](https://github.com/AndelaOSP/shufflebox-frontend/pull/31#discussion_r110131496), it's our current formatting tool.